v22.1/backup-architecture.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Backup Architecture
+summary: Learn about how the process of a CockroachDB backup job works.
+toc: true
+docs_area: manage
+---
+
+CockroachDB backups operate as _jobs_, which are potentially long-running operations that could span multiple SQL sessions. Unlike regular SQL statements, which CockroachDB routes to the [optimizer](cost-based-optimizer.html) for processing, a [`BACKUP`](backup.html) statement will move into a job workflow. A backup job has two phases: [planning](#planning) and [execution](#execution).
+
+The [overview](#overview) section on this page provides an outline of a backup job's process and a glossary listing some common terminology around backup jobs.
+
+For a more detailed explanation of the phases, see [Planning](#planning) and [Execution](#execution).
+
+## Overview
+
+A high-level workflow of a backup:
+
+1. Validate the `BACKUP` statement.
+1. Determine the keys in the [storage layer](architecture/storage-layer.html) to back up.
+1. Write the backup data to the destination storage location.
+1. Write the metadata of the backup job.
+
+The following diagram illustrates the flow from `BACKUP` statement through to a complete backup in cloud storage:
+
+<img src="{{ 'images/v22.1/backup-overview.png' | relative_url }}" alt="A flow diagram representing the process of a backup job from statement through to backup data stored." style="border:0px solid #eee;max-width:100%" />
+
+### Glossary 
+
+Term         | Description
+-------------+-----------------------------
+Coordinator  | The node that claims the job and coordinates the distributed work for the backup job.
```

Comment:
If we did that, I think the structure of this changes to be "Step one is to make a backup job from the backup statement. this requires validating the options and checking permissions, then creating the job record that reflects what to backup, where to back it up, and any other options. After this, if the user requested a "detached" backup the statement is complete, as it has created an uncommitted but otherwise ready-to-run backup job. If they did not request detached, this job is committed, then the statement waits for the job to be started, run and terminate to then return the result of the job execution as the statement's result."